src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/SuperUser/SuperUserSideMenuItem.cs

```diff
@@ -0,0 +1,24 @@
+using Smart.FA.Catalog.Shared.Domain.Enumerations;
+
+namespace Smart.FA.Catalog.Web.Pages.SuperUser;
+
+public class SuperUserSideMenuItem : Enumeration
+{
+    public string Href { get; }
+
+    public static readonly SuperUserSideMenuItem SuperUserTrainingList = new(1, CatalogResources.Trainings, Routes.SuperUserTrainingList);
+
+    protected SuperUserSideMenuItem(int id, string name, string href) : base(id, name)
+    {
+        Href = href;
+    }
+
+    public string? DisplayName()
```

Comment:
Return type doesn't need to be nullable since we control the return value
```suggestion
    public string DisplayName()
```

Reply:
No feedback provided yet for this comment

Reply:
>No feedback provided yet for this comment
Didn't back then you meant I resolved right ?